### DISCUSSION TITLE
[website] Time to get rid of "migrate to v6" ?

### DISCUSSION THREAD

**Discussion by gregfenton** [invertase/react-native-firebase#7485] [General]
Looking at the CHANGELOG, it isn't clear to me what "v5" and "v6" refer to.  RNF is now releasing "v18+".  So does it make sense to still have the "[migrate to v6](https://rnfirebase.io/migrating-to-v6)" page as a top-level link on the website?

**Comment by mikehardy** [comment]
That is a reasonable point. That is some ancient history right there!
